contentcuration/contentcuration/frontend/shared/data/public.js

```diff
@@ -43,7 +43,6 @@ const CONTENT_NODE_FIELD_MAP = {
   node_id: 'id',
   content_id: 'content_id',
   channel_id: 'channel_id',
-  parent: 'parent',
```

Comment:
why is this deleted here?

Reply:
By removing this from the map, it prevents the `parent` from being overwritten in `convertContentNodeResponse`, because it's now manually being set.